docker/steve/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN wget --no-verbose https://github.com/jwilder/dockerize/releases/download/$DO
     && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
     && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
 
-RUN wget -qO- https://github.com/RWTH-i5-IDSG/steve/archive/steve-3.4.5.tar.gz | tar xz --strip-components=1
+RUN wget -qO- https://github.com/steve-community/steve/archive/steve-3.6.0.tar.gz | tar xz --strip-components=1
```

Comment:
I had to do a
```bash
docker-compose build steve
```
before running
```bash
docker-compose up -d
```

Probably a
```bash
docker compose up -d --build
```
would have worked as well

Might be something for the documentation.

Reply:
Yes, for sure. That's usual when using `docker-compose` :slightly_smiling_face:

Yeah, would be good to document. @krealyt